explain.md

```diff
@@ -18,6 +68,209 @@ The user requires the appropriate [privileges](privileges.html) for the statemen
 
 ## Parameters
 
-| Parameter | Description |
+Each `EXPLAIN` statement can only include one *Mode*, but can include any number of *Options*.
+
+| Parameter | Type | Description |
```

Comment:
In the rendered table, the Type column is very narrow. Perhaps try to widen it in the markdown code by adding more dashes underneath its title.